### ISSUE TITLE
Bitcoin only multisig with Passphrase

### ISSUE THREAD

**Issue by G-Sattled** [KeystoneHQ/keystone3-firmware#1431]
Please support the Multisig feature with activated Passphrase. At the moment i can't find it on the roadmap and hope you will add it in near future

**Comment by mildobread** [comment]
I agree. This feature is essential. All other wallets support it. I don’t understand why a passphrase wallet can't be used in a multisig.

**Comment by MattFrom72** [comment]
There has been no response to this yet, but it is also something I am hoping will be implemented in the near future.
